airbyte-webapp/src/components/connection/CatalogTree/streamConfigHelpers/streamConfigHelpers.test.ts

```diff
@@ -283,33 +306,159 @@ describe(`${toggleFieldInPrimaryKey.name}`, () => {
       const mockConfig: AirbyteStreamConfiguration = {
         ...mockStreamConfiguration,
         fieldSelectionEnabled: true,
-        selectedFields: [{ fieldPath: FIELD_ONE }],
+        selectedFields: [{ fieldPath: FIELD_ONE.path }],
       };
 
-      const newStreamConfiguration = toggleFieldInPrimaryKey(mockConfig, FIELD_TWO, 100);
+      const newStreamConfiguration = toggleFieldInPrimaryKey(mockConfig, FIELD_TWO.path, 100);
 
       expect(newStreamConfiguration).toEqual({
-        primaryKey: [FIELD_TWO],
+        primaryKey: [FIELD_TWO.path],
         fieldSelectionEnabled: true,
-        selectedFields: [{ fieldPath: FIELD_ONE }, { fieldPath: FIELD_TWO }],
+        selectedFields: [{ fieldPath: FIELD_ONE.path }, { fieldPath: FIELD_TWO.path }],
       });
     });
 
     it("disables field selection when selected primary key is the last unselected field", () => {
       const mockConfig: AirbyteStreamConfiguration = {
         ...mockStreamConfiguration,
-        primaryKey: [FIELD_ONE],
+        primaryKey: [FIELD_ONE.path],
         fieldSelectionEnabled: true,
-        selectedFields: [{ fieldPath: FIELD_ONE }, { fieldPath: FIELD_TWO }],
+        selectedFields: [{ fieldPath: FIELD_ONE.path }, { fieldPath: FIELD_TWO.path }],
       };
 
-      const newStreamConfiguration = toggleFieldInPrimaryKey(mockConfig, FIELD_THREE, 3);
+      const newStreamConfiguration = toggleFieldInPrimaryKey(mockConfig, FIELD_THREE.path, 3);
 
       expect(newStreamConfiguration).toEqual({
-        primaryKey: [FIELD_ONE, FIELD_THREE],
+        primaryKey: [FIELD_ONE.path, FIELD_THREE.path],
         selectedFields: [],
         fieldSelectionEnabled: false,
       });
     });
   });
 });
+
+describe(`${updateFieldSelected.name}`, () => {
```

Comment:
I've been using this convention:

Function: `#${myFunction.name}`
Component: `<${MyComponent.name} />`

Reply:
Not sure I quite follow here - are you suggesting that I put a `#` before the function name in the test title? Something like this?
```
describe(`#${updateFieldSelected.name}`, () => {
```
I can't find any other usage like this in the codebase, so I think I'm misunderstanding here.